integration/map_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
```

Comment:
Suggestion: Lately, I've been organizing imports like this:

```go
import (
	<golang libs>

	<other libs>

	<aliased imports>

	<_ imports>
)
```

Eg:

```go
import (
	"context"
	"encoding/hex"

	"github.com/google/trillian"
	"github.com/google/trillian/merkle"

	stestonly "github.com/google/trillian/storage/testonly"

	_ "github.com/google/trillian/merkle/coniks"
	_ "github.com/google/trillian/merkle/maphasher"
)
```

I find it cleaner to look at, as blocks become more "uniform" (I saw this in an internal example somewhere). Totally optional.